Src/BiometryService.Uno/BiometryService.Uno.csproj

```diff
@@ -10,7 +10,7 @@
     <Authors>nventive</Authors>
     <Company>nventive</Company>
     <AssemblyName>BiometryService</AssemblyName>
-    <PackageId>BiometryService</PackageId>
+    <PackageId>BiometryService.Uno</PackageId>
```

Comment:
this is considered a breaking change no?

Reply:
if it's considered a breaking change it's for the previous pr

Reply:
The breaking change happened in the previous PR. This is to fix it.